astropy/wcs/tests/test_wtbarr.py

```diff
@@ -43,16 +43,16 @@ def test_wtbarr_print(tab_wcs_2di, capfd):
     s = str(tab_wcs_2di.wcs.wtb[0])
     lines = s.split("\n")
     assert captured.out == s
-    assert "     i: 1" == lines[0]
-    assert "     m: 1" == lines[1]
-    assert "  kind: c" == lines[2]
-    assert "extnam: WCS-TABLE" == lines[3]
-    assert "extver: 1" == lines[4]
-    assert "extlev: 1" == lines[5]
-    assert " ttype: wavelength" == lines[6]
-    assert "   row: 1" == lines[7]
-    assert "  ndim: 3" == lines[8]
+    assert lines[0] == "     i: 1"
+    assert lines[1] == "     m: 1"
+    assert lines[2] == "  kind: c"
+    assert lines[3] == "extnam: WCS-TABLE"
+    assert lines[4] == "extver: 1"
+    assert lines[5] == "extlev: 1"
+    assert lines[6] == " ttype: wavelength"
+    assert lines[7] == "   row: 1"
+    assert lines[8] == "  ndim: 3"
```

Comment:
What's the point of this change??? `if a == b` or `if `b == a`. 🤯

Reply:
The standard is to do ``get_result(x) == expect`` but sometimes, like here, we've done this in a "Yoda" like way of ``expect == get_result(x)``. While (supposed to be) equivalent in Python, it's contrary to how we *think*: we test results against expectations, not expectations against results. The linter can automatically reformat these tests.
There are instances where we really do want ``expect == get_result(x)`` — when we're testing more complex properties of ``__eq__`` and how python can hand the operation off to the right. But in this case we probably want to explicitly note that we are testing these subtle properties. In general ``get_result(x) == expect`` is cleaner.